File: tests/goldens/csharp/valid_minimal.cs
using Xunit;

public class CalculatorTests
{

    // Source: tests/fixtures/valid_minimal.tdd:10
    [Fact]
    public void testAddsTwoNumbers()
    {
        // Input: {'a': 2, 'b': 3}
        var input = new Dictionary<string, object> { { "a", 2 }, { "b", 3 } };
        var result = Calculator.add(input);
        Assert.Equal(5, result);
    }

}

File: tests/goldens/swift/valid_minimal.swift
import XCTest
@testable import calculator

final class CalculatorTests: XCTestCase {

    // Source: tests/fixtures/valid_minimal.tdd:12
    func testAddsTwoNumbers() throws {
        // Input: {'a': 2, 'b': 3}
        let input = Add(a: 2, b: 3)
        let result = add(input: input)
        XCTAssertEqual(result, 5)
    }

}
